data/sql_partition/partition_schedule_a.sql

```diff
@@ -0,0 +1,177 @@
+    contbr_occupation           VARCHAR(38),
+    election_tp                 VARCHAR(5),
+    fec_election_tp_desc        VARCHAR(20),
+    fec_election_yr             VARCHAR(4),
+    election_tp_desc            VARCHAR(20),
+    contb_aggregate_ytd         NUMERIC(14,2),
+    contb_receipt_dt            TIMESTAMP,
+    contb_receipt_amt           NUMERIC(14,2),
+    receipt_tp                  VARCHAR(3),
+    receipt_tp_desc             VARCHAR(90),
+    receipt_desc                VARCHAR(100),
+    memo_cd                     VARCHAR(1),
+    memo_cd_desc                VARCHAR(50),
+    memo_text                   VARCHAR(100),
+    cand_id                     VARCHAR(9),
+    cand_nm                     VARCHAR(90),
+    cand_nm_first               VARCHAR(38),
+    cand_m_nm                   VARCHAR(20),
+    cand_nm_last                VARCHAR(38),
+    cand_prefix                 VARCHAR(10),
+    cand_suffix                 VARCHAR(10),
+    cand_office                 VARCHAR(1),
+    cand_office_desc            VARCHAR(20),
+    cand_office_st              VARCHAR(2),
+    cand_office_st_desc         VARCHAR(20),
+    cand_office_district        VARCHAR(2),
+    conduit_cmte_id             VARCHAR(9),
+    conduit_cmte_nm             VARCHAR(200),
+    conduit_cmte_st1            VARCHAR(34),
+    conduit_cmte_st2            VARCHAR(34),
+    conduit_cmte_city           VARCHAR(30),
+    conduit_cmte_st             VARCHAR(2),
+    conduit_cmte_zip            VARCHAR(9),
+    donor_cmte_nm               VARCHAR(200),
+    national_cmte_nonfed_acct   VARCHAR(9),
+    increased_limit             VARCHAR(1),
+    action_cd                   VARCHAR(1),
+    action_cd_desc              VARCHAR(15),
+    tran_id                     TEXT,
+    back_ref_tran_id            TEXT,
+    back_ref_sched_nm           VARCHAR(8),
+    schedule_type               VARCHAR(2),
+    schedule_type_desc          VARCHAR(90),
+    line_num                    VARCHAR(12),
+    image_num                   VARCHAR(18),
+    file_num                    NUMERIC(7,0),
+    link_id                     NUMERIC(19,0),
+    orig_sub_id                 NUMERIC(19,0),
+    sub_id                      NUMERIC(19,0) NOT NULL,
+    filing_form                 VARCHAR(8) NOT NULL,
+    rpt_tp                      VARCHAR(3),
+    rpt_yr                      NUMERIC(4,0),
+    election_cycle              NUMERIC(4,0),
+    timestamp                   TIMESTAMP,
+    pg_date                     TIMESTAMP,
+    pdf_url                     TEXT,
+    contributor_name_text       TSVECTOR,
+    contributor_employer_text   TSVECTOR,
+    contributor_occupation_text TSVECTOR,
+    is_individual               BOOLEAN,
+    clean_contbr_id             VARCHAR(9),
+    two_year_transaction_period SMALLINT
+);
+
+-- Create the child tables.
+SELECT create_itemized_schedule_partition('a', 1978, 2018);
+SELECT finalize_itemized_schedule_a_tables(1978, 2018);
```

Comment:
I forgot to make a note of this and change it before.  We may want to shift these to environment variables that are made available to the SQL environment, much like `START_YEAR` and `START_YEAR_AGGREGATE` are.  These are managed in the [application config](https://github.com/18F/openFEC/blob/develop/webservices/config.py#L23).

I can also see how this would be confusing though, as it took me a while to remember how these were setup.  Perhaps there's a better way?  Totally fine if that's a separate story for the future, too, and we just add these values in for now.